web/src/backend/verify.ts

```diff
@@ -24,6 +24,12 @@ const KNOWN_ERROR_CODES = [
     detail:
       "The provided proof is invalid and it cannot be verified. Please check all inputs and try again.",
   },
+  {
+    rawMessage: "Root provided in semaphore proof is too old.",
```

Comment:
might be missing a rawCode

Reply:
rawCode is error signature from smart contracts, the root age is enforced at the sequencer level so it doesn't have a similar rawCode

and we don't actually use the rawCode for matching errors in the dev portal side